### DISCUSSION TITLE
table index is nil

### DISCUSSION THREAD

**Discussion by lorcx** [LazyVim/LazyVim#1074] [Q&A]
git clone https://github.com/LazyVim/starter ~/.config/nvim

<pre>
Error detected while processing /home/lx/.config/nvim/init.lua:
Invalid plugin spec {
  checker = {
    enabled = true
  },
  defaults = {
    lazy = false,
    version = false
  },
  install = {
    colorscheme = { "tokyonight", "habamax" }
  },
  performance = {
    rtp = {
      disabled_plugins = { "gzip", "tarPlugin", "tohtml", "tutor", "zipPlugin" }
    }
  },
  spec = { { "LazyVim/LazyVim",
      import = "lazyvim.plugins"
    }, {
      import = "plugins"
    } }
}
E5113: Error while calling lua chunk: ...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/plugin.lua:80: table index is nil

stack traceback:
        ...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/plugin.lua:80: in function 'add'
        ...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/plugin.lua:116: in function 'normalize'
        ...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/plugin.lua:217: in function 'spec'
        ...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/plugin.lua:225: in function 'load'
        /home/lx/.local/share/nvim/lazy/lazy.nvim/lua/lazy/init.lua:45: in function 'setup'
        /home/lx/.config/nvim/lua/config/lazy.lua:9: in main chunk
        [C]: in function 'require'
        /home/lx/.config/nvim/init.lua:2: in main chunk
</pre>

I used neovim 0.9.1 and reported the above error

**Comment by folke** [comment]
Can you remove /home/lx/.local/share/nvim/lazy/lazy.nvim and try again?
That stacktrace does not point to a recent lazy.nvim version

**Reply by lorcx** [reply]
I tried, but it didn't work

**Reply by folke** [reply]
Do you get the same stacktrace after deleting lazy?
